src/storage/btree.c

```diff
@@ -22459,6 +22459,38 @@ btree_rv_data_pack_btid_and_stats (const BTID * btid, long long nulls, long long
   written_size = CAST_BUFLEN (datap - data);
 }
 
+/*
+ * btree_rv_global_unique_stats_commit_dump - dump stored info about global unique stats
+ *
+ */
+void
+btree_rv_global_unique_stats_commit_dump (FILE * fp, int length, void *data)
+{
+  BTID btid;
+  log_unique_stats stats;
+
+  assert (length >= (3 * OR_BIGINT_SIZE) + OR_BTID_ALIGNED_SIZE);
+
+  const char *data_ptr = (const char *) data;
+
+  OR_GET_BTID (data_ptr, &btid);
+  data_ptr += OR_BTID_ALIGNED_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_nulls);
+  data_ptr += OR_BIGINT_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_oids);
+  data_ptr += OR_BIGINT_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_keys);
+  data_ptr += OR_BIGINT_SIZE;
+
+  assert (CAST_BUFLEN (data_ptr - (const char *)data) == length);
+
+  fprintf (fp, "OID = page_id=%d|vol_id=%d|file_id=%d   NEW_STATS = %lld keys, %lld objects, %lld nulls:  %d\n",
```

Comment:
How about replace OID = page_id=%d|vol_id=%d|file_id=%d with BTID = (%d, %d|%d) or BTREE_ID_MSG macro?

Nit:
And when I checked another debug messages, they use "nulls=%lld, oids=%lld, keys=%lld" style.

Reply:
And the last %d is not used.

Reply:
Fixed